src/theme/DocItem/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {HtmlClassNameProvider} from '@docusaurus/theme-common';
+import {DocProvider} from '@docusaurus/theme-common/internal';
+import DocItemMetadata from '@theme/DocItem/Metadata';
+import DocItemLayout from '@theme/DocItem/Layout';
+import ReactMarkdown from 'react-markdown';
+import style from '@site/src/css/markdown-styles.module.css';
+import type {Props} from '@theme/DocItem';
+
+function EOLPage(message: string) {
```

Comment:
function name seems a little odd, considering it's not a page
```suggestion
function EOLMessage(message: string): JSX.Element {
```